File: src/main/java/com/example/debug/engine/model/PerformanceThresholds.java
package com.example.debug.engine.model;

/* loaded from: classes.dex */
public class PerformanceThresholds {
    private int maxSimultaneousTasks = 1;
    private int maxSimultaneousThreads;
    private long threadCheckInterval;

    public PerformanceThresholds() {
    }

    public PerformanceThresholds(int i, long j) {
        this.maxSimultaneousThreads = i;
        this.threadCheckInterval = j;
    }

    public long getThreadCheckInterval() {
        return this.threadCheckInterval;
    }

    public void setThreadCheckInterval(long j) {
        this.threadCheckInterval = j;
    }

    public int getMaxSimultaneousThreads() {
        return this.maxSimultaneousThreads;
    }

    public void setMaxSimultaneousThreads(int i) {
        this.maxSimultaneousThreads = i;
    }

    public int getMaxSimultaneousTasks() {
        return this.maxSimultaneousTasks;
    }

    public void setMaxSimultaneousTasks(int i) {
        this.maxSimultaneousTasks = i;
    }
}

File: src/features/engine/model/performance_thresholds.rs

use serde::{Deserialize, Serialize};

#[derive(Debug, Clone, Serialize, Deserialize)]
pub struct PerformanceThresholds {
    pub max_simultaneous_tasks: i32,
    pub max_simultaneous_threads: i32,
    pub thread_check_interval: i64,
}

impl Default for PerformanceThresholds {
    fn default() -> Self {
        Self {
            max_simultaneous_tasks: 1,
            max_simultaneous_threads: 0,
            thread_check_interval: 0,
        }
    }
}
